micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -1,5 +1,22 @@
 # Changelog 
 
+## [1.8.13]  [16-Dec-2024]
+- Made validations for name, mobile number, and password fields in user profile update screen configurable through MDMS data
```

Markdownlint (0.37.0)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)